### ISSUE TITLE
Add 100 Automations project to the project database

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/VRMS#232]
### Overview
The meeting times on the hackforla website are driven by an VRMS api.  So incoming requests for changes to that page, have to be sent here to be updated manually.  We need to add this new project the output.

### Action Items
- [x] Add meeting time: 
   - [x] Wednesdays: 4:30-5:30 Management Meeting
   - [x] Satudays 2:00pm-4:00pm Team Meeting

### Resources/Instructions
[100 automations project details card](https://raw.githubusercontent.com/hackforla/website/gh-pages/_projects/100-automations.md)
[Hack for LA team meetings page](https://www.hackforla.org/project-meetings)

**Comment by clockwerkz** [comment]
@NivenPrasad Hey! Nice to meet you. :) Feel free to reach me on Slack. I'm currently evaluating our options on the backend..

**Comment by clockwerkz** [comment]
**Progress:** Getting started on this task as it closely relates to #237.
**Blockers:** None
**Availability:** Mon-Fri after 6 PM

**Comment by clockwerkz** [comment]
@NivenPrasad I'm sorry for the delay! For some reason I understood this project to be a code addition to the back end. You just need an entry into the db for your weekly meetings. I'll take care of that this afternoon so it should be up for tomorrow's Management Meeting

**Comment by clockwerkz** [comment]
Ok I have added two separate recurringevents in the database; one for the management team on Wednesdays and one for Saturdays team meeting. I'll be double checking tomorrow to see if the management meeting shows up in the dropdown checkin list..if it doesn't, plz me know!

**Comment by ExperimentsInHonesty** [comment]
Hi @clockwerkz @alex-anakin it doesn't look like its showing up yet.  Here is the data json from the website after we hit the api: https://raw.githubusercontent.com/hackforla/website/gh-pages/_data/vrms_data.json.  It dosen't look the same as the 311 project for example:

**100 Automations entry**
{
"owner": {
"ownerId": "123456"
},
"brigade": "Hack for LA",
"checkInReady": false,
"_id": "5f4602a1f62faf0017d07f6d",
"name": "100 Automations",
"hacknight": "hacknight",
"eventType": "Management Team Meeting",
"description": "Hack for LA seeks to reduce repetitive work in our open source projects and for the open source community.  This project will be both a showcase for the automations and/or microservices that we develop, but also a convenient home for those automations, so that they can be found, forked, and contributed to easily.",
"date": "2020-08-19T23:30:00.000Z",
"hours": 1,
"createdDate": "2020-08-26T06:35:13.510Z",
"updatedDate": "2020-08-26T06:35:13.510Z",
"__v": 0
},
{
"owner": {
"ownerId": "123456"
},
"brigade": "Hack for LA",
"checkInReady": false,
"_id": "5f4602fef62faf0017d07f6e",
"name": "100 Automations",
"hacknight": "hacknight",
"eventType": "Team Meeting",
"description": "Hack for LA seeks to reduce repetitive work in our open source projects and for the open source community.  This project will be both a showcase for the automations and/or microservices that we develop, but also a convenient home for those automations, so that they can be found, forked, and contributed to easily.",
"date": "2020-08-22T21:00:00.000Z",
"hours": 2,
"createdDate": "2020-08-26T06:36:46.692Z",
"updatedDate": "2020-08-26T06:36:46.692Z",
"__v": 0
}

**311 entries**
{
"location": {
"city": "Los Angeles",
"state": "CA",
"country": "USA"
},
"owner": {
"ownerId": "123456"
},
"brigade": "Hack for LA",
"checkInReady": false,
"_id": "5eb72c1a509367be7272cf72",
"hacknight": "Online",
"name": "311 Data Team Meeting",
"eventType": "Team Meeting",
"description": "Team Meeting",
"project": {
"recruitingCategories": [],
"partners": [
"EmpowerLA"
],
"_id": "5eb9e2de3409ffef8194ffde",
"name": "311 Data",
"description": "The 311 Data project seeks to empower local Neighborhood Councils to improve the ideation and analysis of their initiatives using the wealth of publicly available 311 data.",
"githubIdentifier": "190321758",
"projectStatus": "Active",
"location": "DTLA",
"githubUrl": "https://github.com/hackforla/311-data",
"slackUrl": "https://hackforla.slack.com/archives/CJY4R382X",
"googleDriveUrl": "https://drive.google.com/drive/u/0/folders/10_KYe3pbZqiq6reeLA8zDDeIlz-4PxWM",
"googleDriveId": "10_KYe3pbZqiq6reeLA8zDDeIlz-4PxWM",
"hflaWebsiteUrl": "https://www.hackforla.org/projects/311-data",
"videoConferenceLink": "https://us04web.zoom.us/j/8358875318",
"lookingDescription": "Frontend Developers, Backend Developers, Backend Lead, DevOps",
"createdDate": "2020-05-11T23:42:22.028Z",
"__v": 0
},
"date": "2020-05-13T02:00:00.000Z",
"startTime": "2020-05-13T02:00:00.000Z",
"endTime": "2020-05-13T04:00:00.000Z",
"hours": 2,
"createdDate": "2020-05-09T22:18:02.058Z",
"updatedDate": "2020-05-09T22:18:02.058Z",
"__v": 0
}

{
"location": {
"city": "Los Angeles",
"state": "CA",
"country": "USA"
},
"owner": {
"ownerId": "123456"
},
"brigade": "Hack for LA",
"checkInReady": false,
"_id": "5eb73328509367be7272cf78",
"hacknight": "Online",
"name": "311 Data Management Meeting",
"eventType": "Team Meeting",
"description": "Management Meeting",
"project": {
"recruitingCategories": [],
"partners": [
"EmpowerLA"
],
"_id": "5eb9e2de3409ffef8194ffde",
"name": "311 Data",
"description": "The 311 Data project seeks to empower local Neighborhood Councils to improve the ideation and analysis of their initiatives using the wealth of publicly available 311 data.",
"githubIdentifier": "190321758",
"projectStatus": "Active",
"location": "DTLA",
"githubUrl": "https://github.com/hackforla/311-data",
"slackUrl": "https://hackforla.slack.com/archives/CJY4R382X",
"googleDriveUrl": "https://drive.google.com/drive/u/0/folders/10_KYe3pbZqiq6reeLA8zDDeIlz-4PxWM",
"googleDriveId": "10_KYe3pbZqiq6reeLA8zDDeIlz-4PxWM",
"hflaWebsiteUrl": "https://www.hackforla.org/projects/311-data",
"videoConferenceLink": "https://us04web.zoom.us/j/8358875318",
"lookingDescription": "Frontend Developers, Backend Developers, Backend Lead, DevOps",
"createdDate": "2020-05-11T23:42:22.028Z",
"__v": 0
},
"date": "2020-05-15T02:00:00.000Z",
"startTime": "2020-05-15T02:00:00.000Z",
"endTime": "2020-05-15T03:00:00.000Z",
"hours": 1,
"createdDate": "2020-05-09T22:48:08.215Z",
"updatedDate": "2020-05-09T22:48:08.215Z",
"__v": 0
}

**Comment by clockwerkz** [comment]
@ExperimentsInHonesty Yep, I'll look into this again. I suspect the cronjob that's running the events may be erroring out. I need to see if there's any sort of errors being generated on the backend of the live site. I don't have access to this.

The issue at hand is that our data models don't really have any "required" attributes.. so I've gone through and looked at what data the the function that renders events currently needs based on the data that it pulls from each document in the recurringevent entry. On the Development build, I'm able to generate events from what I've inputted into the entries, and in turn, the checkins cronjob will open the event by setting it's checkInReady to true. This week I was able to get the Management Meeting for 100 Automations to show up, but unfortunately it didn't work for the team meeting today. 

In short, I am *potentially* blocked by a) no access to the live site (maybe someone who does have access to it could look for me), and b) we've just updated our development branch so we'll need to roll out these latest changes to the master.

I'll keep investigating this issue for sure!

**Comment by clockwerkz** [comment]
Meeting times are displaying correctly, but the "Meetings" heading is missing from the project page. Would have to look into how the website displays the project/meeting data in the individual project page and see if it's an issue on our end:
![image (25)](https://user-images.githubusercontent.com/11682523/92012761-b83d5980-ed01-11ea-872e-2bb5bbd71277.png)

**Comment by NivenPrasad** [comment]
Thanks @clockwerkz for working on this - looks like all fixed now! Was an issue on the website team side and they've addressed it.

**Comment by Rankazze** [comment]
Confirmed times are posting to HfLA website.  Above is approval from Customer.
